Godeps/Godeps.json

```diff
@@ -6,250 +6,95 @@
 		"./..."
 	],
 	"Deps": [
+		{
+			"ImportPath": "github.com/asaskevich/govalidator",
+			"Comment": "v6-25-g73945b6",
+			"Rev": "73945b6115bfbbcc57d89b7316e28109364124e1"
+		},
 		{
 			"ImportPath": "github.com/beorn7/perks/quantile",
 			"Rev": "3ac7bf7a47d159a033b107610db8a1b6575507a4"
 		},
 		{
 			"ImportPath": "github.com/cloudflare/cfssl/auth",
-			"Comment": "1.3.1-12-g8d76cdf",
+			"Comment": "1.3.1-12-g8d76cdf7",
 			"Rev": "8d76cdf742f917ba01551bf7ca41f8a7f82da15a"
 		},
 		{
 			"ImportPath": "github.com/cloudflare/cfssl/certdb",
-			"Comment": "1.3.1-12-g8d76cdf",
+			"Comment": "1.3.1-12-g8d76cdf7",
 			"Rev": "8d76cdf742f917ba01551bf7ca41f8a7f82da15a"
 		},
 		{
 			"ImportPath": "github.com/cloudflare/cfssl/certdb/dbconf",
-			"Comment": "1.3.1-12-g8d76cdf",
+			"Comment": "1.3.1-12-g8d76cdf7",
```

Comment:
These are spurious diffs; I'd recommend updating your `git`.

Reply:
It's the travis version which is out of date, I'd rather not roll back from 2.17.0.

Will manually revert the changes.

Reply:
(Although didn't we land code to ignore the comments anyway, so is it necessary to make these changes?)

Reply:
> (Although didn't we land code to ignore the comments anyway, so is it necessary to make these changes?)

Yes, in https://github.com/letsencrypt/boulder/commit/0306a933ac3cea6a137ec6ac6f59ec0d8d055e48 - these diffs will be ignored.

Reply:
Ah, I had it backwards. Mainly I didn't want to introduce changes that would be flapping back and forth. Now that we ignore comments, I'm fine with following the modern-Git convention.